src/App.tsx

```diff
@@ -10,9 +11,19 @@ export const App = () => {
 
       <div className="section">
         <div className="container">
-          <h1 className="title">Home Page</h1>
-          <h1 className="title">Page not found</h1>
-          <PeoplePage />
+          <Routes>
+            <Route path="/" element={<Layout />}>
+              <Route path="/" element={<h1 className="title">Home Page</h1>} />
+              <Route path="/home" element={<Navigate to="/" replace />} />
+              <Route path="/people" element={<PeoplePage />}>
+                <Route path=":personSlug" element={<PeoplePage />} />
+              </Route>
+              <Route
+                path="*"
+                element={<h1 className="title">Page not found</h1>}
```

Comment:
Create components `HomePage` and `NotFoundPage`. I don't why exactly, but mentors said so

Reply:
Good point, create component 😄 
Because you HomePage can grow in the future